flink-formats/flink-json/src/main/java/org/apache/flink/formats/json/JsonRowFormatFactory.java

```diff
@@ -79,6 +79,14 @@ public JsonRowFormatFactory() {
                                 schema.ignoreParseErrors();
                             }
                         });
+        descriptorProperties
```

Comment:
Currently the community is moving to the new descriptor api. It not recommended to add new feature for the depricated api.